README.md

```diff
@@ -99,12 +99,19 @@ The repo contains the following components. More libraries are coming soon. ['Wa
 * **System.IO.UnmanagedMemoryStream**. Provides a stream for accessing unmanaged memory as represented by a pointer, 
   as well as an accessor for reading and writing primitive types from unmanaged memory.
 
+* **System.Linq**. Provides the foundation of Language-Integrated Query (LINQ), including LINQ standard query operators 
+  that operate on objects that implement ```IEnumerable<T>```.
+
+* **System.Linq.Queryable**. Provides LINQ standard query operators that operate on objects that implement ```IQueryable<T>```.
+
 * **System.Linq.Parallel**.  Provides a parallelized implementation of LINQ to Objects. "Parallel LINQ" (PLINQ) 
   implements the full set of LINQ standard query operators as well as additional operators specific to parallel operations.
 
 * **System.Numerics.Vectors**. Provides a set of basic vector types that leverage single instruction, 
   multiple data (SIMD) CPU instructions. See our [recent][simd-post-1] [announcements][simd-post-2] for more details.
 
+* **System.ObjectModel**. Provides classes that can be used in the object model of a reusable library.
```

Comment:
Thanks, @akoeplinger.  I used the NuGet package description, and added more libraries to the readme using the NuGet package description, or at least using them as the basis.